web/src/ChooseArea.svelte

```diff
@@ -77,7 +95,7 @@
 
     <div style="columns: 2">
       {#each listAllFiles() as [boundary, list]}
-        <div class="group">
+        <div class="file-group">
```

Comment:
I'm keeping these here for now, because I don't think I'll have time to make the file loading flow nicer. Or maybe we just totally hide this now, because it's still possible to click an area, then load a file.

Reply:
I commented this out as it's the ultimate position following other changes, but left the HTML present as I assumed you would still need this for now.